### ISSUE TITLE
issue with double quotes when splitting a string

### ISSUE THREAD

**Issue by giacomociti** [eyereasoner/eye#102]
I can use `string:join` to split a string:

```turtle
@prefix string: <http://www.w3.org/2000/10/swap/string#> .
@prefix : <http://ex.org/>.

{ 
    (?result ",") string:join "a,\"quoted\",c" .
} 
=> 
{ 
    :result :is ?result
}.
```
but, in case of double quotes, I get an invalid result:

`:result :is ("a" ""quoted"" "c"). `

**Comment by josd** [comment]
Thanks for your observation and this should now be fixed in `EYE v9.7.7 (2024-02-10)` and give
```
:result :is ("a" "\"quoted\"" "c").
```

**Comment by giacomociti** [comment]
thanks
